cache/s3/s3.go

```diff
@@ -141,6 +144,19 @@ func New(config dict.Dicter) (cache.Interface, error) {
 		return nil, err
 	}
 
+	// If a Proxy/Sidecar/etc.. is used, the Host header needs to
+	// be fixed to allow Request Signing to work.
+	// More info: https://github.com/aws/aws-sdk-go/issues/1473
+	req_signing_host := DefaultReqSigningHost
```

Comment:
minor nit here, but can you change the var naming to match the rest of the conventions in this func? For example:

`req_signing_host` -> `reqSigningHost`. Minor detail, but I would like to keep the conventions consistent this close together.

Reply:
Definitely! I used Python syntax in a Go file, my brain confused the two :) Hope that now works, lemme know @ARolek!

Reply:
Haha, no worries. Looks good! I'm just waiting for the CI and then I will merge.